transcom/mymove
# B 20654 in

## [B-20654](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A982774)
2nd shot
## Summary
Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the milmovelocal app
2. click view test harness scenarios
3. click 'MakeMoveReadyForEdi'
4. in the payload, find the payment_request_numver in the payment_request object

<img width="1296" alt="image" src="https://github.com/user-attachments/assets/78097c84-7ab8-49b8-8baf-352d4f87f5a4">

5. go to your terminal and run ` go run ./cmd/generate-payment-request-edi --payment-request-number 7551-1584-1
`
should see a success message 
6. go to db and see that there is a record in payment_request_edi_uploads and uploads table


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
